marklogic-client-api/src/main/java/com/marklogic/client/expression/PlanBuilderBase.java

```diff
@@ -464,6 +464,18 @@ public interface PlanBuilderBase {
      */
     TransformDefinition transformDefinition(String path);
 
+    /**
+     * Build a schema definition for use with {@code validateDoc}.
+     *
+     * @return a new {@code SchemaDefExpr}
+     */
+    SchemaDefExpr schemaDefinition(String kind);
```

Comment:
I just renamed TransformDefinition/transformDefinition to TransformDef/transformDef to be consistent with the other types in the "client.expression" package. So I think we should do SchemaDef/schemaDef and then errorDispositionDef is good to go.

Reply:
Hi Rob, that does not look right to me since all other files in expression package end with "Expr". Do you want to rename transformDef to TransformDefExpr?

Reply:
Hmm - after reading the package-info.java file and looking at the existing "Expr" classes, my thought is to stick with "Def" as the suffix. That's because the existing "Expr" classes seem different in nature - they all seem to be collections of functions, whereas our "Def" classes are defining sets of config options.

But as long as the 3 names are consistent, it's easy to attach "Expr" later if we decide that's the right move. Good question for Erik on Tuesday.